packages/mobile/src/pincode/PincodeSet.test.tsx

```diff
@@ -134,4 +136,36 @@ describe('Pincode', () => {
     await flushMicrotasksQueue()
     expect(getByText('pincodeSet.pinsDontMatch')).toBeDefined()
   })
+
+  it('navigates to the Settings screen after successfully changing PIN', async () => {
+    const { getByTestId, rerender } = render(
+      <Provider store={mockStore}>
+        <PincodeSet {...mockScreenProps} />
+      </Provider>
+    )
+
+    // Create pin
+    mockPin.split('').forEach((number) => fireEvent.press(getByTestId(`digit${number}`)))
+    jest.runAllTimers()
+    await flushMicrotasksQueue()
+    expect(mockScreenProps.navigation.setParams).toBeCalledWith({
+      isVerifying: true,
+      changePin: true,
```

Comment:
`setParams` is never called with `changePin` so you'll want to remove this line

Reply:
ah i see i might have just been confused that this had to be set again